touch-core-internal/Controllers/EmployeeController.cs

```diff
@@ -80,6 +80,28 @@ public virtual async Task<IActionResult> GetByNameAsync(string username = null)
                 serviceResponse.UpdateResponseStatus($"Employee does not exist", false);
                 return this.NotFound("");
             }
+        } 
+        
+        [Route("{email?}"), HttpGet]
+        public virtual async Task<IActionResult> GetByEmailAsync(string email = null)
```

Comment:
if this endpoint is driven by email, then I don't think we need email to be optional. Here email should be compulsory. Any specific reason for email as an optional parameter?

Reply:
Makes sense, removed the optional check